src/componets/LoginPage.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { useFormik } from "formik";
+import { loginSchema } from "../schemas/index";
+import logoLogin from "../assets/img/logo-login.svg";
+
+const initialValues = {
+  email: ""
+};
+
+function LoginPage() {
+  const { values, errors, touched, handleBlur, handleChange, handleSubmit } =
+    useFormik({
+      initialValues,
+      validationSchema: loginSchema,
+      onSubmit: (actions) => {
+        actions.resetForm();
+      }
+    });
+
+  return (
+    <div className="w-100">
+      <div className="row">
+        <div className="col-lg-4 pe-lg-0 d-lg-flex">
+          <div className="login-bg"></div>
+        </div>
+        <div className="col-lg-8 ps-lg-0 bg-white d-flex align-items-center">
+          <div className="login-form-content">
+            <div className="login-form">
+              {/* <!-- Logo --> */}
+              <a href="#" className="text-center login-logo">
+                <img src={logoLogin} alt="AWAYDAY" className='login-logo animation-rotate' />
+              </a>
+              {/* <!-- Login Form --> */}
+              <form className="login-form-wrap" onSubmit={handleSubmit}>
```

Comment:
You can also create separate login form component, same as we did in deposyt admin